tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -1,10 +1,9 @@
----
 apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   name: extract-binaries-from-image
   labels:
-    app.kubernetes.io/version: "1.1.0"
+    app.kubernetes.io/version: "1.2.0"
```

Comment:
Can we go to 2.0.0 here?

Reply:
Yes. I wasn't sure if you wanted 2.0.0 or 1.2.0 here since this is backwards compatible. Will change to 2.0.0 in next commit